python/perf-kernels/tools/tune_gemm/tune_gemm.py

```diff
@@ -72,13 +73,15 @@ def get_full_tuning_space():
                             for num_stages in num_stage_range:
                                 for waves_per_eu in waves_per_eu_range:
                                     for matrix_instr_nonkdim in matrix_instr_nonkdim_range:
-                                        for kpack in kpack_range:
-                                            configs.append({
-                                                'BLOCK_SIZE_M': block_m, 'BLOCK_SIZE_N': block_n, 'BLOCK_SIZE_K':
-                                                block_k, 'GROUP_SIZE_M': group_m, 'SPLIT_K': split_k, 'num_warps':
-                                                num_warps, 'num_stages': num_stages, 'waves_per_eu': waves_per_eu,
-                                                'matrix_instr_nonkdim': matrix_instr_nonkdim, 'kpack': kpack
-                                            })
+                                        for sched_variant in sched_variants:
```

Comment:
Since you are here, can you replace the nested for loops with itertools.product?

Reply:
Yes, sure. Done!